### PR TITLE
[10.0][MIG] product_by_supplier

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
The changes included in this migration are:
* Basic migration actions (odoo tags, bump version, templates...).
* Update the README.
* Fix moved external IDs.

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
cc  @eficent

**Review by gmeficent — APPROVED** [review]
Functional review :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
It works as expected :+1:

File: product_by_supplier/README.rst [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
-Product by supplier info
+========================
+Product by Supplier Info
 ========================
 
-This module add a new menu item 'Product by suppliers' in Purchases -> Products.
-It allows for users to be able to view a compiled list of products supplied by the supplier.
-This information is the corresponding to the supplier list on each product.
+This module add a new menu item 'Product by suppliers' in Purchases ->
```
Comment (pedrobaeza) [inline product_by_supplier/README.rst]: s/add/adds

File: product_by_supplier/README.rst [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+Products. It allows for users to be able to view a compiled list of products
```
Comment (pedrobaeza) [inline product_by_supplier/README.rst]: Remove `for`

File: product_by_supplier/README.rst [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
+supplied by the supplier. This information is the corresponding to the
```
Comment (pedrobaeza) [inline product_by_supplier/README.rst]: s/the supplier/their supplier

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Little redaction bugs.

File: product_by_supplier/README.rst (hunk 3 above)
Comment (LoisRForgeFlow) [reply product_by_supplier/README.rst]: I've attended these mistakes.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**